internal/pkg/docker/orchestrator/orchestrator.go

```diff
@@ -281,7 +281,8 @@ func (o *Orchestrator) containerID(name string) string {
 // Containers within a Task can talk to each other on localhost
 // and are stopped and started as a group.
 type Task struct {
-	Containers map[string]ContainerDefinition
+	Containers   map[string]ContainerDefinition
+	PauseSecrets map[string]string
```

Comment:
Why do we need this to be a separate field as opposed to use `t.Containers["pause"].Secrets` 🤔

Reply:
Oh nvm sorry i didn't realize we don't really have a "pause" container in `Containers`.